### PR TITLE
Make all trips option available (whereever the data exists) for non-prod branches

### PR DESCRIPTION
**Pull request by usr110**
This is in reference to the issue created here: https://github.com/npct/pct-shiny/issues/705

@Robinlovelace, are you up to test it first, and then merge?

I think it would be wise to merge to `master` and then `prod` afterwards. What do you think?

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Thanks Ali, will test when I get a chance on work desktop - likely next week now.

**Comment by Robinlovelace** [comment]
Update: just looked at the code, looks good to me, will test on a local copy of prod. @nikolai-b can you see any issues with this? Good fix by the looks of it @usr110 many thanks.

File: regions_www/m/server.R [hunk 1]
```diff
@@ -41,6 +41,9 @@ if (!on_server) {
 
 repo_sha <-  as.character(readLines(file.path(interface_root, "repo_sha")))
 
+# Check if the branch is production or not
+is_prod <- ifelse(system("git symbolic-ref -q --short HEAD", intern = T) == "production", TRUE, FALSE)
```
Comment (nikolai-b) [inline regions_www/m/server.R]: The code isn't inside a git repo in production so sadly this doesn't work:

```bash
root@npct0:/var/shiny/pct-shiny# git symbolic-ref -q --short HEAD
fatal: Not a git repository (or any of the parent directories): .git
```
Which I think you spotted first https://github.com/npct/pct-shiny/commit/76cd62e40e103e29f64d946f6f96bb10aa5ea710

I believe we are after
```R
# Check if we are on the production server (npt followed by any number of digits (only) is a prod machine)
production_branch <- grepl("npt\\d*$", Sys.info()["nodename"])
```
(from deb05c6)

Comment (usr110) [reply regions_www/m/server.R]: Very good point @nikolai-b  - thanks!
Will make the required changes.  So glad you checked it before it got merged.
I had a recollection that we did something like this before, but couldn't find the commit - may need to learn more about git, it seems.

Comment (nikolai-b) [reply regions_www/m/server.R]: Sure!

I used [git log -S](https://git-scm.com/docs/git-log#Documentation/git-log.txt--Sltstringgt) with `git log -S production` to list relevant commits then I just read the commit messages and used `git show`

Comment (usr110) [reply regions_www/m/server.R]: Noted, thanks!

**Comment by usr110** [comment]
Thanks @nikolai-b  for the merge!

**Pull request merged**